runtime/Makefile

```diff
@@ -20,7 +20,7 @@ test: all
 testCoverage: $(LIB) $(BCLIB)
 	$(MAKE) -C tests testCoverage
 
-$(LIB): $(OBJS)
+$(LIB): $(TAGGED_PTR_OBJ) $(OBJS)
```

Comment:
We want taggedToPtr to be inlined by the C file.

This means we need to do

- compile .c to .bc
- llvm-link the .bc with tagged-ptr.ll
- llc the result into .o

Reply:
With the change de1dc58 I am getting an error as shown below. I am currently looking into that.

```
clang-11  -o ./string_hash string_hash.c ../balrt.a
/usr/bin/ld: ../balrt.a: error adding symbols: archive has no index; run ranlib to add one
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Reply:
Fixed that issue in be597b8